src/runtime/styles.ts

```diff
@@ -67,6 +67,8 @@ export const addStyle = (styleContainerNode: any, cmpMeta: d.ComponentRuntimeMet
           styleContainerNode.insertBefore(styleElm, styleContainerNode.querySelector('link'));
         }
 
+        styleElm.innerHTML += SLOT_FB_CSS;
```

Comment:
For my own understanding, why do we add this here?

Reply:
This is how we style the `slot-fb` element for CE builds. This appends the `slot-fb` styles to the style element in the DOM. Added a code comment as well

Reply:
This probably ties directly into my question above (GH won't let me link it since I haven't submitted the review yet):

> Double checking - why should we always add this style to DOM, even if a component doesn't have any slots? I think that's still lost on me, even with the added comment

How do we know this is a `slot-fb` element?

Reply:
You're right, we shouldn't. For some reason didn't realize we actually know if we need to apply this for each component. Updated to wrap this in a conditional based on `CMP_FLAGS.hasSlotRelocation`